File: Core/SmartGrid/Models/Podesavanja.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace SmartGrid.Models
{
    public class Podesavanja
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [Required]
        public bool Obavezna { get; set; }
        [Required]
        public bool Errorvidljive { get; set; }
        [Required]
        public bool Infovidljive { get; set; }
        [Required]
        public bool Warningvidljive { get; set; }
        [Required]
        public bool Successvidljive { get; set; }
    }
}

File: clientside/src/app/entities/podesavanja.ts
export class Podesavanja {

    id:number;
    obavezna:boolean;
    errorvidljive:boolean;
    infovidljive:boolean;
    warningvidljive:boolean;
    successvidljive:boolean;

    constructor(obavezna:boolean,errorvidljive:boolean,infovidljive:boolean,warningvidljive:boolean,successvidljive:boolean)
    {
        this.obavezna=obavezna;
        this.errorvidljive=errorvidljive;
        this.infovidljive=infovidljive;
        this.warningvidljive=warningvidljive;
        this.successvidljive=successvidljive;
        
    }

}
